packages/contracts/test/contracts/L2/predeploys/OVM_ETH.spec.ts

```diff
@@ -0,0 +1,73 @@
+import { expect } from '../../../setup'
+
+/* External Imports */
+import { ethers } from 'hardhat'
+import { ContractFactory, Contract, Signer } from 'ethers'
+
+describe('OVM_ETH', () => {
+  let signer1: Signer
+  let signer2: Signer
+  before(async () => {
+    ;[signer1, signer2] = await ethers.getSigners()
+  })
+
+  let Factory__OVM_ETH: ContractFactory
+  before(async () => {
+    Factory__OVM_ETH = await ethers.getContractFactory('OVM_ETH')
+  })
+
+  let OVM_ETH: Contract
+  beforeEach(async () => {
+    OVM_ETH = await Factory__OVM_ETH.deploy()
+  })
+
+  describe('transfer', () => {
+    it('should revert', async () => {
+      await expect(
+        OVM_ETH.transfer(await signer2.getAddress(), 100)
+      ).to.be.revertedWith(
+        'OVM_ETH: ERC20 features are disabled pending further community discussion.'
+      )
+    })
+  })
+
```

Comment:
Same comment as above - do we even need to have these functions exist? We can just delete them + these tests for now